src/main/java/org/alliancegenome/curation_api/jobs/executors/BulkLoadJobExecutor.java

```diff
@@ -49,6 +49,8 @@ public class BulkLoadJobExecutor {
 	@Inject GeneMolecularInteractionExecutor geneMolecularInteractionExecutor;
 	@Inject GeneGeneticInteractionExecutor geneGeneticInteractionExecutor;
 	@Inject ParalogyExecutor paralogyExecutor;
+	@Inject
+	GeneExpressionExecutor geneExpressionExecutor;
```

Comment:
Injection should just be on one line - don't know why not flagged by checkstyle

Reply:
Fixed